node/service/src/service.rs

```diff
@@ -468,7 +466,10 @@ fn start_consensus(
 		relay_chain_slot_duration,
 		proposer,
 		collator_service,
+		#[cfg(any(feature = "frequency-testnet", feature = "frequency-local", feature = "frequency-lint-check"))]
 		// Very limited proposal time.
+		authoring_duration: Duration::from_millis(2000),
```

Comment:
Q: is this `2000`? Is seems like it should be but just wanted to make sure since in [here](https://wiki.polkadot.network/docs/maintain-guides-async-backing)  it says the following which might have changed.

> Increase authoring_duration from 500 milliseconds to 1500

Reply:
Thanks for taking a thorough look. These docs are slightly outdated compared to what was actually implemented in the [code](https://github.com/paritytech/polkadot-sdk/blob/9cdf3d999e9de7996495c891af7085f55acc695a/templates/parachain/node/src/service.rs#L214).